src/views/Tweets.vue

```diff
@@ -77,17 +77,21 @@ export default class Tweets extends Vue {
   show_tweet(tweet : any) {
     this.selected = tweet;
 
-    this.analyse(tweet.text, "sentiment", this.analysis);
-    this.analyse(tweet.text, "category", this.analysis);
-    this.analyse(tweet.text, "entities", this.analysis);
+    this.analyse(tweet.text, "sentiment", this.analysis, false);
```

Comment:
Pass `this.summaryDone` or `this.detailDone` instead of boolean, this parameter should be optional.